server/verify.go

```diff
@@ -176,6 +179,7 @@ func VerifyAttestation(attestation *pb.Attestation, opts VerifyOpts) (*pb.Machin
 
 		proto.Merge(machineState, celState)
 		proto.Merge(machineState, state)
+		proto.Merge(machineState, teeState)
```

Comment:
We should avoid proto.Merge where possible as it can override existing values.

Reply:
Done.